server.ts

```diff
@@ -66,9 +73,33 @@ if (args.startSimple) {
     createSimpleServer(args);
 }
 
+function getBuildsLinks(userAgent: string = ''): Array<{ url: string; text: string }> {
+    const result = [];
+    const platform = userAgent.includes('Electron') ? 'desktop' : 'web';
```

Comment:
Knowledge about available platforms is dispersed across the code. To prevent possible difficulties with it we might need to consider its concentration in one place.